apps/aechannel/src/aesc_fsm.erl

```diff
@@ -2652,6 +2728,22 @@ check_inband_msg(#{ channel_id := ChanId
 check_inband_msg(_, _) ->
     {error, chain_id_mismatch}.
 
+check_client_reconnect_tx(SignedTx, #data{ opts = #{ reconnect_security := none } } = D) ->
```

Comment:
Shouldn't this parameter be documented somewhere?

Reply:
Not sure if we want to keep it in its current form after the conference. :)
I would prefer a reconnect check based on a session token instead.